data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,265 @@ event "remnant: pantica void sprites"
+			choice
+				`	"Thanks for the explanation."`
+					goto astralmoving
+				`	"Kind of like a funeral, with friends and family gathering to see you off?"`
+			`	"Oh, no. I said my farewells to my friends and relatives before we left. Those who gathered to see me off today were more of a representation of Remnant society. I only recognized a few of them, but they took the time to come down to the <ship> to see me before I left. They came to see me, and wish me well, so that no matter what challenges I face, I will know that the Remnant remember me and appreicate what I am doing." Her gestures fade into contemplative stillness for a moment, then resume motion.`
+			label astralmoving
+			`	She gestures towards the airlock door. "Lead the way, Captain."`
+				accept
+	on enter "Sagittarius A*"
+		conversation
+			`As the energy ripples from the Jump Drive fade, the spectacular void that is Sagittarius A* takes shape before you. Compared to the vast darkness of space that you are accustomed to, reality here feels almost like a color negative: Even with the automatic filtering in your windows, space is incredibly bright with the densely packed stars that provide a backdrop for Sagittarius A*. The moment of awe is shattered by your alarms warning you of your impending doom as the <ship> begins to fall inwards towards the black hole. You quickly activate your anti-gravity system, which, although vastly under-powered for dealing with the immense pull of a black hole, does provide a degree of protection so long as you maintain a safe distance from it.`
+			`	After a few minutes of just absorbing the spectacle before you, Dawn pulls herself away from the cockpit windows and scurries down to the lab. Leaving you to your thoughts, she quickly has the lab set up and running a variety of recordings, test, and observation equipment. After a few hours of work, she reappears on the bridge.`
+			`	"Things are proceeding smoothly, Captain. All the sensors are working nominally and we are creating extensive archives of astronomical data as well. Another few hours and we should have all the data we need."`
+			choice
+				`	"Good work. What kind of data are you recording?`
+				`	"Sounds good. Just let me know when you are ready to leave."`
+					goto astraldone
+			`	She smiles, and replies. "It might be easier to ask what data we are leaving out. The data we are collecting broadly falls into three data sets: Data on Sagittarius A* itself, data on its environment, and observations of the surrounding galaxy from here. Obviously, we cannot go take samples of Sagittarius A* itself, nor take direct measurements of most of its properties, but we can get data from increasingly minute distances from the event horizon. We can then add this data to that gathered by previous expeditions, note the differences, and use it in our modeling."`
```

Comment:
```suggestion
			`	She smiles, and replies. "It might be easier to ask what data we are leaving out. The data we are collecting broadly falls into three data sets: data on Sagittarius A* itself, data on its environment, and observations of the surrounding galaxy from here. Obviously, we cannot go take samples of Sagittarius A* itself, nor take direct measurements of most of its properties, but we can get data from increasingly minute distances from the event horizon. We can then add this data to that gathered by previous expeditions, note the differences, and use it in our modeling."`
```